### PR TITLE
Feature done about and contact page

### PR DESCRIPTION
**Pull request by LAEEQ18**
About Page for review.

Contact Page for review.

privacy policy done in index.js

### CODE REVIEW

File: src/utils/font-awesome.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  faInstagram
+)
+//   library. for adding from libray we use
+// syntax  {/* <FontAwesomeIcon icon={[{"fab","linkedin"]} size="2x" /> */} fa for font awesom and b for brand;
+// without library syntax is <FontAwesomeIcon icon={faGithub} size="2x" />
+/* <FontAwesomeIcon icon={faGithub} size="2x" />
+<FontAwesomeIcon icon={faSlack} size="2x" />
+<FontAwesomeIcon icon={faYoutube} size="2x" />
+<FontAwesomeIcon icon={faEnvelope} size="2x" />
+*/
```
Comment (hanif-arif) [inline src/utils/font-awesome.js:30]: remove

File: src/pages/contacts.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import { Row, Col } from 'antd'
+import Layout from '../components/layout'
+import cont from '../images/contact.png'
+import './contact.css'
+import '../utils/font-awesome'
+import 'antd/dist/antd.css'
+
+const contacts = () => (
```
Comment (hanif-arif) [inline src/pages/contacts.js:9]: Components in React have PascalCase

File: src/pages/about.js [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+  padding: 3px;
+  width: 100%;
+  color: white;
+  background-color: black;
+  text-align: center;
+`
+
+// function changeBackground(e) {
+//   e.target.style.background = 'grey'
+// }
```
Comment (hanif-arif) [inline src/pages/about.js:22]: remove

File: src/pages/contact.css [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+@media (max-width: 767px) {
+  .contact-option {
+    border: 0;
+    padding: 20px 15px;
+  }
+
+  .contact-center {
+    display: none;
+  }
+}
```
Comment (hanif-arif) [inline src/pages/contact.css:41]: use styled components

File: package.json [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    "lint:fix": "eslint . --fix"
+  },
+  "dependencies": {
+    "@fortawesome/fontawesome-free": "^5.15.1",
+    "@fortawesome/fontawesome-svg-core": "^1.2.32",
+    "@fortawesome/free-brands-svg-icons": "^5.15.1",
+    "@fortawesome/free-solid-svg-icons": "^5.15.1",
+    "@fortawesome/react-fontawesome": "^0.1.12",
+    "antd": "^4.8.2",
+    "babel-plugin-styled-components": "^1.11.1",
```
Comment (hanif-arif) [inline package.json:21]: move to devDependency

File: package.json [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+    "gatsby-plugin-styled-components": "^3.3.14",
+    "gatsby-source-contentful": "^3.0.6",
+    "gatsby-source-filesystem": "^2.3.37",
+    "gatsby-transformer-sharp": "^2.5.20",
+    "global": "^4.4.0",
+    "i": "^0.3.6",
+    "jquery": "^3.5.1",
+    "popper.js": "^1.16.1",
+    "react": "^16.14.0",
+    "react-bootstrap": "^1.4.0",
```
Comment (hanif-arif) [inline package.json:35]: remove react-bootstrap

File: src/components/Footer.js [hunk 7]
```diff
@@ -0,0 +1,129 @@
+import { Link } from 'gatsby'
+import React from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { Row, Col, Menu } from 'antd'
+import '../utils/font-awesome'
+import intel from '../images/intelf.png'
+
+function Footer() {
+  return (
```
Comment (hanif-arif) [inline src/components/Footer.js:9]: be consistent use Arrow function or functions.

File: src/components/layout.css [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+  line-height: 2;
+}
+
+@media screen and (min-width: 992px) {
+  .images {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    column-gap: 1rem;
+  }
+}
```
Comment (hanif-arif) [inline src/components/layout.css:22]: Move to styled components

File: src/pages/about.js [hunk 9]
```diff
@@ -0,0 +1,227 @@
+import React from 'react'
+import { Row, Col } from 'antd'
+import styled from 'styled-components'
+import Layout from '../components/layout'
+import about from '../images/about.png'
+import pic from '../images/image1.jpeg'
+import pic1 from '../images/image2.jpeg'
+
+// import styles from '../components/about.module.css'
+import 'antd/dist/antd.css' // or 'antd/dist/antd.less'
```
Comment (hanif-arif) [inline src/pages/about.js:10]: you only need to import this once

File: src/components/layout.js [hunk 10]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import Navbar from './Navbar'
+// import Footer from './Footer'
+import Navba from './hed'
+import './layout.css'
```
Comment (hanif-arif) [inline src/components/layout.js:5]: import antd css here

File: src/components/layout.js [hunk 11]
```diff
@@ -0,0 +1,15 @@
...
+import Navbar from './Navbar'
+// import Footer from './Footer'
+import Navba from './hed'
+import './layout.css'
+
+const layout = ({ children }) => (
+  <>
+    <Navbar />
+    <main> {children} </main>
+    <Navba />
```
Comment (hanif-arif) [inline src/components/layout.js:11]: Footer should be here

**Review by hanif-arif — CHANGES_REQUESTED** [review]

File: src/components/Footer.js [hunk 12]
```diff
@@ -0,0 +1,129 @@
...
+        </Col>
+        <Col span={5}>
+          <h2
+            style={{ fontSize: '19px', fontWeight: 'bold', paddingTop: '40px' }}
+          >
+            Get In touch
+          </h2>
+          <div style={{ padding: '1px' }}>
+            <a className="icons" href="https://twitter.com/intelsnap_inc">
+              <FontAwesomeIcon icon={['fab', 'twitter']} size="2x" />
```
Comment (hanif-arif) [inline src/components/Footer.js:90]: open all get in touch links in new tab

File: src/components/hed.js [hunk 13]
```diff
@@ -0,0 +1,159 @@
...
+
+const Navba = () => (
+  <Menu
+    theme="light"
+    mode="horizontal"
+    defaultSelectedKeys={['1']}
+    style={{
+      width: '100%',
+      background: 'grey',
+      marginTop: '30px',
```
Comment (hanif-arif) [inline src/components/hed.js:16]: fix the header to stay on top of the page all the time

**Review by hanif-arif — CHANGES_REQUESTED** [review]

File: package.json (hunk 5 above)
Comment (LAEEQ18) [reply package.json:21]:
```suggestion
    {
	"resource": "/c:/Users/naeem/Desktop/css/react js/GatsbyJs/intel/website/src/components/Navbar.js",
	"owner": "eslint",
	"code": {
		"value": "import/no-extraneous-dependencies",
		"target": {
			"$mid": 1,
			"external": "https://github.com/benmosher/eslint-plugin-import/blob/v2.22.0/docs/rules/no-extraneous-dependencies.md",
			"path": "/benmosher/eslint-plugin-import/blob/v2.22.0/docs/rules/no-extraneous-dependencies.md",
			"scheme": "https",
			"authority": "github.com"
		}
	},
	"severity": 8,
	"message": "'antd' should be listed in the project's dependencies, not devDependencies.",
	
}'@fortawesome/fontawesome-svg-core' should be listed in the project's dependencies, not devDependencies.
'@fortawesome/free-brands-svg-icons' should be listed in the project's dependencies, not devDependencies.
'@fortawesome/free-brands-svg-icons' should be listed in the project's dependencies, not devDependencies.

File: src/components/Footer.js (hunk 7 above)
Comment (LAEEQ18) [reply src/components/Footer.js:9]: done

File: src/components/layout.js (hunk 11 above)
Comment (LAEEQ18) [reply src/components/layout.js:11]: one footer is plain and Navba is current footer

File: src/components/hed.js (hunk 13 above)
Comment (LAEEQ18) [reply src/components/hed.js:16]: done

**Pull request closed without merging**